data_generation/utils/secret_manager_utils.py

```diff
@@ -0,0 +1,14 @@
+import ast
+from google.cloud import secretmanager
+
+
+def get_secret_value_Dict(project_id="", secret_id="", version_id="latest"):
```

Comment:
try except needed, or somehow we need to know secret manager failed if we give wrong inputs. The code should not give exception. It should fail gracefully